pages/paths/[id].jsx

```diff
@@ -72,49 +73,55 @@ const PathDetailPage = () => {
   }
 
   return (
-    <Container>
-      <Typography.Title className="mt-0 mb-16" level={3}>
-        {pathData.name}
-      </Typography.Title>
-      <Row gutter={[48, 48]}>
-        <Col xs={24} lg={12}>
-          <Typography.Title className="mt-0 mb-8" level={4}>
-            Path
-          </Typography.Title>
-          {markdownContent && (
+    <>
+      <Meta
+        title={pathData.name}
+        description={pathData.description}
+        url={pathData.service.url}
```

Comment:
this does't seem correct, probably should contain build domain + paths/{id}
<img width="597" alt="image" src="https://github.com/user-attachments/assets/639d32bf-1d1e-4645-88b2-c68924908ef4">